pybamm/models/submodels/porosity/sei_reaction_driven_porosity.py

```diff
@@ -0,0 +1,61 @@
+#
+# Class for reaction driven porosity changes
+#
+import pybamm
+from .base_porosity import BaseModel
+
+
+class SeiFull(BaseModel):
+    """Full model for reaction-driven porosity changes
+
+    Parameters
+    ----------
+    param : parameter class
+        The parameters to use for this submodel
+
+
+    **Extends:** :class:`pybamm.porosity.BaseModel`
+    """
+
+    def __init__(self, param):
+        super().__init__(param)
+
+    def get_fundamental_variables(self):
+
+        eps_n = pybamm.standard_variables.eps_n
+        eps_s = pybamm.standard_variables.eps_s
+        eps_p = pybamm.standard_variables.eps_p
+        variables = self._get_standard_porosity_variables(eps_n, eps_s, eps_p)
+
+        return variables
+
+    def get_coupled_variables(self, variables):
+        j_sei = variables["Negative electrode sei interfacial current density"]
+        C_sei_eps = pybamm.sei_parameters.C_sei_eps
+
+        deps_n_dt = C_sei_eps * j_sei
+        deps_s_dt = pybamm.FullBroadcast(
+            0, "separator", auxiliary_domains={"secondary": "current collector"}
+        )
+        # for the moment is not needed but could be used for positive SEI
+        deps_p_dt = pybamm.FullBroadcast(0, "positive electrode",
+                                         "current collector")
+
+        variables.update(
+            self._get_standard_porosity_change_variables(
+                deps_n_dt, deps_s_dt, deps_p_dt
+            )
+        )
+
+        return variables
```

Comment:
Still not convinced this warrants a whole separate submodel. All you would need to do to reuse the full submodel is to add these lines to `standard_parameters_lithium_ion.py`:
```python3
beta_surf_n = pybamm.Scalar(0)
beta_surf_p = pybamm.Scalar(0)
```
By the way, just so you are aware, `deps_dt` comes into the `dce_dt` equation (see e.g. `full_diffusion.py`) so a changing porosity will affect the electrolyte concentration (make the electrolyte more concentrated) - again, this comes from the lead-acid models